torch/ao/quantization/fx/prepare.py

```diff
@@ -438,12 +442,16 @@ def _get_target_activation_dtype_for_node(
                 and (not input_act_is_dynamic)
             ) else torch.float
 
+        is_general_tensor_value_op = \
+            (qhandler is not None and qhandler.is_general_tensor_value_op())
```

Comment:
optional nit: I think "input_output_share_observers" is clearer than "is_general_tensor_value" in general. Can we change this in the qhandler as well?

Reply:
yeah sure, I can change that as well

Reply:
actually, let's do it later since we may deprecate the fx graph mode quant code base